src/validation.c

```diff
@@ -828,9 +829,9 @@ lyd_validate_must(const struct lyd_node *node, int val_opts)
         break;
     case LYS_RPC:
     case LYS_ACTION:
-        if (val_opts & LYD_VALOPT_INPUT) {
+        if (val_opts & LYD_VALIDATE_OP_RPC) {
```

Comment:
Enum should not be checked with `&` so it would be better to separate these into another parameter.

Reply:
true, changed